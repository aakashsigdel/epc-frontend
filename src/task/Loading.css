@keyframes spinner {
  0% { transform: rotate(0deg); }
  100% { transform: rotate(360deg); }
}

.loader {
  margin: auto;
  border: 5px solid lightgray;
  border-top: 5px solid #e88b8b;
  border-radius: 50%;
  width: 40px;
  height: 40px;
  animation: spinner 0.5s linear infinite;
}
