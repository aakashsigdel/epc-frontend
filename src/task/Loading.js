import React from 'react';

import './Loading.css';

const Display = props =>
  <div className="loader" />;

export default Display;

