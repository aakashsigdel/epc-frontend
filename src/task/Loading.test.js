import React from 'react';
import Loading from './Loading';

import renderer from 'react-test-renderer';

it('should render <Loading /> correctly', () => {
  const component = renderer.create(<Loading />)
  const snapshotTree = component.toJSON();

  expect(snapshotTree).toMatchSnapshot();
});



