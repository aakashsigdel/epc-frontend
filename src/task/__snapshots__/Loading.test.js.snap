// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <Loading /> correctly 1`] = `
<div>
  Loading...
</div>
`;
